terraform/fec_rds.tf

```diff
@@ -16,6 +16,42 @@ provider "aws" {
   region = "${var.region}"
 }
 
+resource "aws_iam_role" "monitoring_role" {
+  name = "AmazonRDSEnhancedMonitoringRoleTF"
+
+  assume_role_policy = <<EOF
```

Comment:
It looks like this should be the role policy, not the assume role policy--the first is what the role is allowed to do, and the second is who is allowed to assume the role. I think you want to move this policy to a separate `aws_iam_role_policy` resource and associate the policy resource with the role: https://www.terraform.io/docs/providers/aws/r/iam_role_policy.html.